gatsby-config.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
     {
       resolve: `gatsby-mdx`,
       options: {
+        globalScope: `
+          import ResponsiveEmbed from "react-responsive-embed";
```

Comment:
this is nice approach for "short codes". Here's some context: https://www.christopherbiscardi.com/post/towards-shortcodes-for-gatsby-sites